instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -279,7 +281,7 @@ def _extract_parameters(args, kwargs):
     else:
         # In httpx < 0.20.0, handle_request receives the parameters separately
         method = args[0]
-        url = args[1]
+        url = httpx.URL(args[1])
```

Comment:
Was this a bug previously?

Reply:
I don't think so. The expected type is `typing. Tuple[bytes, bytes, typing.Optional[int], bytes]`. The else statement is just to please httpx older versions regarding parameter extraction, but in those versions httpx.URL also [supports this signature](https://github.com/encode/httpx/blob/0.18.2/httpx/_models.py#L122). What I'm doing here is also returning a httpx.URL for those versions as well. I don't think it's a bug, just did that to avoid doing `httpx.URL` again in `_apply_request_client_attributes_to_span`.

Reply:
I can keep it as `url = args[1]` and do the parser later just to get the URL attributes like host and etc.

Reply:
Opted to keep as-is and do httpx.URL again when generating attributes.